packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -592,7 +614,14 @@ if (require.main === module) {
     process.exit(1);
   }
 
-  Poll(null, process.env.PORT, process.env.SPOKE_URL, process.env.CUSTOM_NODE_URL, hubConfig).then(() => {});
+  Poll(
+    null,
+    process.env.PORT,
+    process.env.SPOKE_URL,
+    process.env.CUSTOM_NODE_URL,
+    hubConfig,
+    process.env.SPOKE_URLS
```

Comment:
this is a backward compatible change, if `SPOKE_URLS` is not defined everything should run as normal

Reply:
Is this env already set up in the RL bots?

Reply:
no it has not been added

Reply:
this is cool as it means that the existing tests pass but it would be good to add tests for this! you should be able to see how things are set up [here](https://github.com/UMAprotocol/protocol/blob/master/packages/serverless-orchestration/test/ServerlessHub.js) and add a test case where we use multiple `SPOKE_URLS`